client/app/bundles/course/assessment/submission/components/GetHelpChatPage/ConversationArea.tsx

```diff
@@ -0,0 +1,111 @@
+import { FC } from 'react';
+import { Typography } from '@mui/material';
+
+import LoadingEllipsis from 'lib/components/core/LoadingEllipsis';
+import { useAppSelector } from 'lib/hooks/store';
+import useTranslation from 'lib/hooks/useTranslation';
+
+import { getLiveFeedbackChatsForAnswerId } from '../../selectors/liveFeedbackChats';
+import translations from '../../translations';
+import { ChatSender } from '../../types';
+
+interface ConversationAreaProps {
+  onFeedbackClick: (linenum: number) => void;
+  answerId: number;
+}
+
+const ConversationArea: FC<ConversationAreaProps> = (props) => {
+  const { onFeedbackClick, answerId } = props;
+  const liveFeedbackChats = useAppSelector((state) =>
+    getLiveFeedbackChatsForAnswerId(state, answerId),
+  );
+  const { t } = useTranslation();
+
+  const isRequestingLiveFeedback = liveFeedbackChats?.isRequestingLiveFeedback;
+  const isPollingLiveFeedback = liveFeedbackChats?.pendingFeedbackToken;
+
+  const isRenderingSuggestionChips =
+    !isRequestingLiveFeedback && !isPollingLiveFeedback;
+
+  if (!liveFeedbackChats) return null;
+
+  const justifyAlign = (isStudent: boolean, isError: boolean): string => {
```

Comment:
note: in flex terminology, justify is about left-right positioning, while align is about top-bottom positioning.

Reply:
changed to `justifyPosition`